### DISCUSSION TITLE
[skywalking traceid'problem]  i use skywalking  python agent 1.0 in python'microservice( flask+gunicorn) but it's parent'microservice(java skywalking agent) can not transfer traceid to child'microservice(python)

### DISCUSSION THREAD

**Discussion by wuweixing2018** [apache/skywalking#10475] [Q&A]
my parent'microservice use sprint Boot'dev and install skywalking java agent ,this services'request can show his request trace'information,but his child'microservice with python dev(flask+gunicorn) and install  skywalking python agent v1.0，but request trace can not transfer to python'microservice request  ?

**Comment by Superskyyy** [comment]
Sorry but this really doesnt provide too much information for debugging. The reason of breaking trace is caused by entry span not recognizing the passed down trace is in header. Can you investigate what header where received on the Python side (flask request handler)? E.g. were they intercepted and removed by other things?

**Comment by wuweixing2018** [comment]
ok,thankyou for feedbacking  for me .
